src/styles/settings/stars.scss

```diff
@@ -0,0 +1,34 @@
+.stars {
+  margin-top: 16px;
+  display: flex;
+  justify-content: space-between;
+
+  &__stars {
+    display: flex;
+    gap: 4px;
+  }
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__stars &__star:nth-child(1n) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
The selector `&__stars &__star:nth-child(1n)` is equivalent to `&__stars &__star`, which targets all children. If the intention is to target every star, consider simplifying the selector to `&__stars &__star`.